tests/test_stt_whisper_service.py

```
@@ -0,0 +1,129 @@
+import os
+import tempfile
+from types import SimpleNamespace
+
+import pytest
+
+# Patch OpenAI before importing service to avoid side-effects
+import importlib, types
```

Ruff (0.12.2)

`importlib` imported but unused

Remove unused import: `importlib`

(F401)